pkg/graphql/client.go

```diff
@@ -66,6 +69,13 @@ func (c *ClientImpl) ExecuteQuery(ctx context.Context, query string, variables m
 
 	res, err := c.httpClient.Do(req)
 	if err != nil {
+		var urlErr *url.Error
+		if errors.As(err, &urlErr) {
+			var x509Err x509.UnknownAuthorityError
+			if errors.As(urlErr, &x509Err) {
```

Comment:
I think you were matching this property initially

```suggestion
			if errors.As(urlErr.Err, &x509Err) {
```